<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="12236" started-at="2020-07-31T06:47:09Z" finished-at="2020-07-31T06:47:21Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="12236" started-at="2020-07-31T06:47:09Z" finished-at="2020-07-31T06:47:21Z">
      <class name="com.orangehrm.tests.LoginTest">
        <test-method status="PASS" signature="intialize()[pri:0, instance:com.orangehrm.tests.LoginTest@13deb50e]" name="intialize" is-config="true" duration-ms="692" started-at="2020-07-31T06:47:08Z" finished-at="2020-07-31T06:47:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intialize -->
        <test-method status="PASS" signature="openBrowser(java.lang.reflect.Method)[pri:0, instance:com.orangehrm.tests.LoginTest@13deb50e]" name="openBrowser" is-config="true" duration-ms="3887" started-at="2020-07-31T06:47:09Z" finished-at="2020-07-31T06:47:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.orangehrm.tests.LoginTest.verifyLoginWithValidCredentials() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="PASS" signature="openApplication()[pri:0, instance:com.orangehrm.tests.LoginTest@13deb50e]" name="openApplication" is-config="true" duration-ms="232" started-at="2020-07-31T06:47:13Z" depends-on-methods="com.orangehrm.tests.BaseTest.openBrowser" finished-at="2020-07-31T06:47:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApplication -->
        <test-method status="FAIL" signature="verifyLoginWithValidCredentials()[pri:0, instance:com.orangehrm.tests.LoginTest@13deb50e]" name="verifyLoginWithValidCredentials" duration-ms="6660" started-at="2020-07-31T06:47:13Z" finished-at="2020-07-31T06:47:20Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Verify Welcome Message expected [Welcome Adminn] but found [Welcome Admin]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Verify Welcome Message expected [Welcome Adminn] but found [Welcome Admin]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.orangehrm.tests.LoginTest.verifyLoginWithValidCredentials(LoginTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginWithValidCredentials -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.orangehrm.tests.LoginTest@13deb50e]" name="closeBrowser" is-config="true" duration-ms="1437" started-at="2020-07-31T06:47:20Z" finished-at="2020-07-31T06:47:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginWithValidCredentials status=FAILURE method=LoginTest.verifyLoginWithValidCredentials()[pri:0, instance:com.orangehrm.tests.LoginTest@13deb50e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.orangehrm.tests.LoginTest@13deb50e]" name="tearDown" is-config="true" duration-ms="272" started-at="2020-07-31T06:47:21Z" finished-at="2020-07-31T06:47:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.orangehrm.tests.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
